src/app/about/page.client.tsx

```diff
@@ -0,0 +1,151 @@
+'use client';
+import TeamBioCard from '@/components/team-bio-card/team-bio-card';
+import {
+  Box,
+  Button,
+  Pagination,
+  PaginationItem,
+  Typography,
+} from '@mui/material';
+import Container from '@mui/material/Container';
+import Script from 'next/script';
+import React, { useState } from 'react';
+import './about.scss';
+
+interface TeamMember {
+  name: string;
+  job_title: string;
+  bio?: string;
+  linkedin_link: string;
+  github_link?: string;
+  role: string[];
+  profile_image_link: string;
+  brief_descriptions?: string;
+}
+
+interface AboutPageProps {
+  teamMemberData: TeamMember[];
+}
+
+export default function About({ teamMemberData }: AboutPageProps) {
+  const [filter, setFilter] = useState<string>('All Members');
+  const [page, setPage] = useState<number>(1);
+
+  const teamMemberJson = teamMemberData.map((teamMember: TeamMember) => ({
+    '@context': 'https://schema.org',
+    '@type': 'Person',
+    name: teamMember.name,
+    jobTitle: teamMember.job_title,
+    description: teamMember.bio || '',
+    url: 'https://www.codeforbc.ca/about',
+    image: teamMember.profile_image_link
+      ? `https://www.codeforbc.ca${teamMember.profile_image_link}`
+      : '',
+    sameAs: [teamMember.linkedin_link, teamMember.github_link || ''],
+  }));
+
+  const roles = [
+    'All Members',
+    'Executive',
+    'Product/UIUX',
+    'Developers',
+    'Business Development',
+    'Data',
+  ];
+
+  const filterMembers = (role: string) => {
+    if (role === 'All Members') return teamMemberData;
```

Comment:
```suggestion
    if (role === roles[0] return teamMemberData;
```

Similar to my previous suggestion